Sources/Twift+Lists.swift

```diff
@@ -103,6 +103,35 @@ extension Twift {
   public func deleteList(_ listId: List.ID) async throws -> TwitterAPIData<DeleteResponse> {
     return try await call(route: .list(listId), method: .DELETE, expectedReturnType: TwitterAPIData.self)
   }
+    
+    /// Enables the authenticated user to create a new List.
+    /// - Parameters:
+    ///  - name: List name (required)
+    ///  - description: Description for the list (optional)
+    ///  - private: Whether created list is private and viewable only by the authenticated user or publicly viewable
+    /// - Returns: A response object containing the name and the ID of the list.
+    public func createList(name: String, description: String? = "", isPrivate: Bool = false) async throws -> TwitterAPIData<CreatedListResponse> {
+        
+        let body: [String : Any] = [
+            "name": name,
+            "description": description ?? "",
+            "private": isPrivate
+        ]
+        let serializedBody = try JSONSerialization.data(withJSONObject: body)
+        return try await call(route: .createList,
+                              method: .POST,
+                              body: serializedBody,
+                              expectedReturnType: TwitterAPIData.self)
+    }
+    
+    /// A response object pertaining to list created
+    /// - Returns:
+    ///  - id: If the returned response object contains an id, response is considered successfull
+    ///  - name: If List name was returned, response is considered successfull
```

Comment:
These comments should be written directly above their respective struct properties so that they show up in Xcode’s quick help and the generated GitHub wiki docs.

Reply:
Oh wow, had no idea 😅